cmd/reports-controller/main.go

```diff
@@ -205,7 +200,6 @@ func main() {
 	flagset.BoolVar(&aggregateReports, "aggregateReports", true, "Enable or disable aggregated policy reports.")
 	flagset.BoolVar(&policyReports, "policyReports", true, "Enable or disable policy reports.")
 	flagset.BoolVar(&validatingAdmissionPolicyReports, "validatingAdmissionPolicyReports", false, "Enable or disable validating admission policy reports.")
-	flagset.IntVar(&reportsChunkSize, "reportsChunkSize", 1000, "Max number of results in generated reports, reports will be split accordingly if there are more results to be stored.")
```

Comment:
This is a breaking change, we probably need to follow the deprecation process.

Reply:
Keeping the flag and drop a warning ?

Reply:
I wonder why this wasn't deprecated in 1.11 🤔

Reply:
>Keeping the flag and drop a warning ?

Yes.

Reply:
And we need to deprecate it in docs https://kyverno.io/docs/installation/customization/#container-flags.

Reply:
Done

Reply:
Docs PR, please.